packages/client/src/components/ExportButton.tsx

```diff
@@ -26,6 +35,13 @@ const ExportButton = () => {
 			<InfoMessage message={'not logged in'} status={RequestStatus.Failed} />
 		);
 	}
+
+	const transcriptFormatDescription: Record<TranscriptFormat, string> = {
+		srt: 'SRT format (Time Coded)',
```

Comment:
another formatting suggestion: 
could we make this `SRT (time-coded)` or `SRT (with time codes)`